Deployments behind a load-balancer (#57)
multipath-tcp/mptcp.dev: load-balancer.md

  v4/v6 IP address and/or port that are specific to this server.
- A L4 load-balancer should route traffic to this specific IP and/or port to the
  right server.

In other words, on Linux, it means that each server should:
- set the [`net.mptcp.allow_join_initial_addr_port`](https://docs.kernel.org/networking/mptcp-sysctl.html)
  sysctl knob to `0`
- add a `signal` MPTCP endpoint with a dedicated IP address and/or port:
  ```
  ip mptcp endpoint add <public IP address> dev <interface> [ port NR  ] signal
  ```

{: .note}
A stateful load-balancer could compute the MPTCP receiver's token from its key
exchanged in the connection request (`MP_CAPABLE`), and route additional
subflows to the same server by identifying the receiver's token from the join
request (`MP_JOIN`). Be careful that there is a risk of token collision, and
such load-balancer should handle the case where multiple end-servers are using
the same token for active MPTCP connections.

## CDNs

Supporting MPTCP would be beneficial for the users, to be able to easily benefit
from MPTCP: seamless handovers, best network selection, and network aggregation.

> Here is a checklist dedicated to CDNs owners to support MPTCP:

Reviewer: ```suggestion
Here is a checklist for CDN owners implementing MPTCP support:
```
